File: Algorithms/Java/448-find_all_numbers_disappeared_in_an_array.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {


        HashSet<Integer> mySet = new HashSet<>();
        ArrayList<Integer> list = new ArrayList<>();

        for(int i: nums){
            mySet.add(i);
        }

        for(int i = 1; i <= nums.length; i++){
            if(!mySet.contains(i)){
                list.add(i);
            }

        }

       return list;
        



    }
}

File: Algorithms/python/448-find_all_numbers_disappeared_in_an_array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        for n in nums:
            v = abs(n) - 1
            nums[v] = -abs(nums[v])
        

        res = []

        for i in range(len(nums)):
            if nums[i] > 0:
                res.append(i + 1)
        return res



        